CONTRIBUTING_JS.md

```diff
@@ -135,6 +157,18 @@ A `gh-pages` branch will be created, and this should be published to via the Git
 
 This makes API docs available, and the types are linked through to from other modules published by aegir.
 
+aegir can also update the `About` section of a README.md based on the `@packageDocumentation` of your entry file via the `aegir docs` command.
+
+#### Front-end and UI
+
+##### React
+
+For projects utilizing React, adhere to the following standards:
+
+1. Component Naming: Use kebab-case for component names (e.g., `my-component`). This is different than idiomatic React, which uses PascalCase.​
+2. File Extensions: Use .tsx for files containing React components to leverage TypeScript's JSX support.​
+3. State Management: Prefer using React's built-in state management features. If additional state management libraries are needed, please ensure they are well-justified, align with the project's architecture, and are well-typed. 
```

Comment:
I agree redux-bundler proved to be a flop. Either way, I wouldn't prescribe this on an org level.

Reply:
We should advise against redux-bundler at the org level because it breaks type-safety.